tidb-cloud/migrate-from-oracle-using-aws-dms.md

```diff
@@ -0,0 +1,148 @@
+---
+title:  Migrate from Amazon RDS for Oracle to TiDB Cloud Serverless Tier Using AWS DMS
+summary: Learn how to migrate data from Amazon RDS for Oracle into TiDB Cloud using AWS Database Migration Service (AWS DMS).
+---
+
+# Migrate from Amazon RDS for Oracle to TiDB Cloud Serverless Tier Using AWS DMS
+
+This document describes a step-by-step example of how to migrate data from Amazon RDS for Oracle to [TiDB Cloud Serverless Tier](https://tidbcloud.com/console/clusters/create-cluster) using AWS Database Migration Service (AWS DMS).
+
+If you are interested in learning more about TiDB and AWS DMS, you can find some useful links as follows:
+
+- [TiDB Cloud](https://docs.pingcap.com/tidbcloud/)
+- [TiDB Developer Guide](https://docs.pingcap.com/tidbcloud/dev-guide-overview)
+- [AWS DMS Documentation](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_GettingStarted.html)
+
+## Why use AWS DMS?
+
+AWS Database Migration Service (AWS DMS) is a cloud service that makes it possible to migrate relational databases, data warehouses, NoSQL databases, and other types of data stores.
+
+If you want to migrate data from from heterogeneous databases, such as PostgreSQL, Oracle, and SQL Server to TiDB Cloud, it is recommended to use AWS Database Migration Service (AWS DMS).
+
+## Deployment architecture
+
+At a high level, follow the following steps:
+
+1. Set up source Amazon RDS for Oracle.
+2. Set up the target [TiDB Cloud (Serverless) Tier](https://tidbcloud.com/console/clusters/create-cluster).
+3. Set up data migration (full load) using AWS DMS.
+
+The following diagram illustrates the high-level architecture.
+
+![Architecture](/media/tidb-cloud/aws-dms-from-oracle-to-tidb-0.png)
+
+## Prerequisites
+
+You might need the following prerequisites:
+
+- [AWS DMS Prerequisites](/tidb-cloud/migrate-from-mysql-using-aws-dms.md#Prerequisites)
```

Comment:
link to here: https://github.com/pingcap/docs/pull/11438/files#diff-929b11aba15553b7b3f1dc934fc150cf892cf8d394c7ff77104e89c8bad95b45R18